oshdb-tool/etl/src/main/java/org/heigit/bigspatialdata/oshdb/tool/importer/util/long2long/LongToLongMap.java

```diff
@@ -1,7 +0,0 @@
-package org.heigit.bigspatialdata.oshdb.tool.importer.util.long2long;
-
-public interface LongToLongMap {
```

Comment:
was this one unused? it is fine to drop it @rtroilo ?

Reply:
same as with the other comment: this is just mitakenly flagged as "deleted + recreated" by git/github instead of the actual "renamed" operation.

Reply:
sorry for the noise